src/app/components/listing/details/surveillance/surveillance-view.jsx

```diff
@@ -0,0 +1,180 @@
+import React, { useState, useEffect } from 'react';
+import { ThemeProvider, makeStyles } from '@material-ui/core/styles';
+import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+} from '@material-ui/core';
+
+import { ChplTooltip } from '../../../util/chpl-tooltip';
+/* eslint-disable import/no-cycle */
+import { getAngularService } from '.';
+/* eslint-enable import/no-cycle */
```

Comment:
Not required, but here if you want it:

```suggestion
import { getAngularService } from '.'; // eslint-disable-line import/no-cycle
```